lib/rake-tasks-mysql/rake_task.rb

```diff
@@ -73,15 +73,15 @@ def parse_mysqlrestore_args(args)
     STDOUT.puts "==> Applying the mysqldump from #{args[:filename]}. Copying the file to the container:\n\n"
 
     docker = services_from_args(services: %w[mysql])
-    docker.upload(args[:filename], args[:filename])
+    container_path = docker.upload(args[:filename], args[:filename])
     if $?.to_i != 0
       STDERR.puts "\n\n==> The dump could not be copied to the container\n\n"
       exit $?.to_i
     end
 
     STDOUT.puts "\n\n==> Copied the dump to the container. Applying to the database:\n\n"
 
-    command = 'gzip --decompress --stdout ' + Shellwords.escape(args[:filename]) + ' | mysql -u"$MYSQL_USER" -p"$MYSQL_PASSWORD" "$MYSQL_DATABASE"'
+    command = 'gzip --decompress --stdout ' + container_path + ' | mysql -u"$MYSQL_USER" -p"$MYSQL_PASSWORD" "$MYSQL_DATABASE"'
```

Comment:
We should probably still Shellwords.escape() the returned filename as `File.basename(destination)` doesn't remove or escape spaces/special shell chars.